### DISCUSSION TITLE
Blured frame

### DISCUSSION THREAD

**Discussion by vindhya124** [dlemstra/Magick.NET#1277] [Help]
Hi Team,

I need to share an image to facebook from my website. Facebook accepts images which is of 1.91:1 ratio so i need a create an image similar to below which avoids cropping when shared on facebook. Is there a way we can achieve blurred borders using Magick.net library?

**Comment by dlemstra** [comment]
It is probably a better idea to ask your question here: https://github.com/ImageMagick/ImageMagick/discussions and then translate a command line example to C#.

**Reply by vindhya124** [reply]
Thank you. Is there a way I can close this discussion?

**Reply by dlemstra** [reply]
There is no close options for discussions. It would be nice if you would post how you resolved this for the next person who reads this.

**Reply by vindhya124** [reply]
I have achieved using Imagesharp library and below are the steps.


I resized the image to 1150 width and 600 height, blurred the background and overlayed my 800*600 image.
Below is code.
image1.Mutate(i => i.Resize(800, 600, KnownResamplers.Lanczos3));

image.Mutate(i => i.Resize(1150, 600, KnownResamplers.Lanczos3).BokehBlur().DrawImage(image1, new Point(180,0), opacity: 1f).Brightness(1.0f));

Below is the facebook debugger tool where we can test how our feed is shared.
https://developers.facebook.com/tools/debug/

And below is the image i am sharing to facebook.

![image](https://user-images.githubusercontent.com/38023844/194955317-4dbd7f05-ae92-410a-ab29-6f17051869b3.png)
